sunpy/net/scraper.py

```diff
@@ -75,6 +74,7 @@ class Scraper:
     def __init__(self, pattern, **kwargs):
         pattern = pattern.format(**kwargs)
         timepattern = pattern
+        self.directories = []
```

Comment:
Why was this added?

Reply:
Would it be possible to patch the mock/path the function that returns the directory list?